examples/oft-aptos/aptos.layerzero.config.ts

```diff
@@ -0,0 +1,161 @@
+                        msgType: 1,
+                        optionType: ExecutorOptionType.LZ_RECEIVE,
+                        gas: 1000, // gas limit in wei for EndpointV2.lzReceive
+                        value: 0, // msg.value in wei for EndpointV2.lzReceive
+                    },
+                    {
+                        msgType: 2,
+                        optionType: ExecutorOptionType.LZ_RECEIVE,
+                        gas: 1000, // gas limit in wei for EndpointV2.lzCompose
+                        value: 0, // msg.value in wei for EndpointV2.lzCompose
+                    },
+                    // {
+                    //     msgType: 1,
+                    //     optionType: ExecutorOptionType.NATIVE_DROP,
+                    //     amount: 0, // amount of native gas token in wei to drop to receiver address
+                    //     receiver: '0x0000000000000000000000000000000000000000',
+                    // },
+                ],
+                sendLibrary: '0xbe533727aebe97132ec0a606d99e0ce137dbdf06286eb07d9e0f7154df1f3f10',
+                receiveLibraryConfig: {
+                    // Required Receive Library Address on Aptos
+                    receiveLibrary: '0xbe533727aebe97132ec0a606d99e0ce137dbdf06286eb07d9e0f7154df1f3f10',
+                    // Optional Grace Period for Switching Receive Library Address on Aptos
+                    gracePeriod: BigInt(0),
+                },
+                // Optional Receive Library Timeout for when the Old Receive Library Address will no longer be valid on Aptos
+                receiveLibraryTimeoutConfig: {
+                    lib: '0xbe533727aebe97132ec0a606d99e0ce137dbdf06286eb07d9e0f7154df1f3f10',
+                    expiry: BigInt(696969669),
+                },
+                sendConfig: {
+                    executorConfig: {
+                        maxMessageSize: 10000,
+                        // The configured Executor address on Aptos
+                        executor: '0xeb514e8d337485dd9ce7492f70128ef5aaa8c34023866e261a24ffa3d61a686d',
+                    },
+                    ulnConfig: {
+                        // The number of block confirmations to wait on Aptos before emitting the message from the source chain.
+                        confirmations: BigInt(300),
+                        // The address of the DVNs you will pay to verify a sent message on the source chain.
+                        // The destination tx will wait until ALL `requiredDVNs` verify the message.
+                        requiredDVNs: ['0xd6f420483a90c7db5ce2ec12e8acfc2bfb7b93829c9e6a3b0760bca330be64dd'],
+                        // The address of the DVNs you will pay to verify a sent message on the source chain.
+                        // The destination tx will wait until the configured threshold of `optionalDVNs` verify a message.
+                        optionalDVNs: ['0xd6f420483a90c7db5ce2ec12e8acfc2bfb7b93829c9e6a3b0760bca330be64dd'],
```

Comment:
we don't usually provide optional dvns by default in other examples, we probably shouldnt add optional dvn here

Reply:
wip for equivalent testing

Reply:
Agreed. Will remove

Reply:
@DanL0 do you mean we should just fully remove it or have an empty array?